.eslintrc.js

```diff
@@ -10,6 +10,7 @@ module.exports = {
     // The `ecmaVersion` should align to the supported features of our target runtimes (browsers / nodejs / others)
     // Consult with: https://kangax.github.io/compat-table/es2016plus/
     ecmaVersion: 2017,
+    project: ["./tsconfig.base.json", "./tsconfig.json"],
```

Comment:
should not these `parserOptions.project` be specified under the `[*.ts]` override?